packages/eslint-mdx/package.json

```diff
@@ -30,19 +30,19 @@
     "eslint": ">=8.0.0"
   },
   "dependencies": {
-    "acorn": "^8.10.0",
+    "acorn": "^8.11.2",
     "acorn-jsx": "^5.3.2",
     "espree": "^9.6.1",
-    "estree-util-visit": "^1.2.1",
-    "remark-mdx": "^2.3.0",
-    "remark-parse": "^10.0.2",
-    "remark-stringify": "^10.0.3",
+    "estree-util-visit": "^2.0.0",
+    "remark-mdx": "^3.0.0",
+    "remark-parse": "^11.0.0",
+    "remark-stringify": "^11.0.0",
     "synckit": "^0.8.5",
-    "tslib": "^2.6.1",
-    "unified": "^10.1.2",
-    "unified-engine": "^10.1.0",
-    "unist-util-visit": "^4.1.2",
+    "tslib": "^2.6.2",
+    "unified": "^11.0.4",
+    "unified-engine": "^11.1.1",
+    "unist-util-visit": "^5.0.0",
     "uvu": "^0.5.6",
```

Comment:
using the latest release of node there's an option to drop `uvu` in favor of `node:test` https://nodejs.org/api/test.html
Which is even faster, a stable part of node core, and works well with ESM

Reply:
Maybe a new PR?

Reply:
We are actually using `uvu/assert` only, maybe it can be replaced by `devlop`. For testing, we're using `jest` now, and will migrate to `vitest` later.